java/src/org/openqa/selenium/grid/data/YesSlotMatcher.java

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.grid.data;
```

Comment:
This should go where the tests are.

Reply:
Yes, I know, but I could not manage to tell bazel to import it 
I ended up with circular dependencies so if you have any clue, I'll apply it

Reply:
It does not work
```
java/test/org/openqa/selenium/grid/node/config/NodeOptionsTest.java:702: error: cannot find symbol
        .isExactlyInstanceOf(YesSlotMatcher.class);
                             ^
  symbol:   class YesSlotMatcher
  location: class NodeOptionsTest
Target //java/test/org/openqa/selenium/grid/node/config:NodeOptionsTest failed to build
```

![image](https://github.com/SeleniumHQ/selenium/assets/18628677/7c05d766-55f1-42e7-ac41-ac4e0eb02feb)

I got it working by creating an other package that contains only the YesSlotMatcher
![image](https://github.com/SeleniumHQ/selenium/assets/18628677/e6e05464-e10a-4936-bcbd-d5c763ee67f2)
Is that ok for you ?